interview-questions.md

```diff
@@ -7,44 +7,76 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: To add a foreign key as a developer if i forgot i would use db:migrate to go back and create the foreign key.
 
   Researched answer:
+  If you forgot to assign a foreign key, you can generate a migration in order to update your database. The syntax will be $ rails generate migration to update. Then you would run $ rails db:migrate after generating a migration.
```

Comment:
Definitely the starting steps. How would you update the change definition? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?